doc/gbs_tutorials.rst

```diff
@@ -2,6 +2,6 @@ Applications layer tutorials
 ============================
 
 .. customgalleryitem::
-    :tooltip: To be filled
-    :description: :doc:`tutorials_gbs/run_tutorial_example`
-    :figure: ./_static/planted.png
\ No newline at end of file
+    :tooltip: Sampling
+    :description: :doc:`tutorials_gbs/run_tutorial_sample`
+    :figure: ./_static/thumbs/code.png
```

Comment:
The code.png is a placeholder while we decide on a better image, right?

Reply:
Hmmm, I'd rather have a colourful, eye-catching thumbnail, like in the PL tutorials. How about this picture of one of the chips? 
![chip](https://user-images.githubusercontent.com/40441298/66855103-a10e4580-ef50-11e9-82a7-0a5ed9e1f02f.png)

Reply:
I got this from Zach, but there are official pictures from the Xanadu style guide. We could potentially crop and compress if the file is too large

Reply:
The chip is also good though, e.g.,
![Xanadu-18](https://user-images.githubusercontent.com/49409390/66861334-d28d0e00-ef5c-11e9-9632-ec3e83ededf6.jpg)

Reply:
Decided:
![image](https://user-images.githubusercontent.com/49409390/66929477-16395380-f001-11e9-8d2c-72b64aed1607.png)

Reply:
That image is highly misleading. It is a single photon giving the impression that you are doing standard Boson Sampling.

Reply:
🤔 Ah the idea was more of counting photons at the output.
Any suggestions on a good image?

Reply:
You could put a Gaussian instead, but it is not nearly as "pretty". When you say counting photons you mean threshold detectors, right?